mlos_bench/mlos_bench/launcher.py

```diff
@@ -481,7 +525,6 @@ def _load_scheduler(self, args_scheduler: Optional[str]) -> Scheduler:
                 config={
                     "experiment_id": "UNDEFINED - override from global config",
                     "trial_id": 0,
-                    "config_id": -1,
```

Comment:
don't remove it - it's a placeholder for config ID passed from the external configuration (e.g., when we run a benchmark trial for existing config)

Reply:
I worked it out.  It had to do with the schema validation.  We only allow config_id >= 0 or else absent config_id.  For now, I just added code to remove that from config copy prior to validation to avoid the issue.

Reply:
I am afraid that if we omit the `config_id` parameter here and specify one in the command line, that value won't be propagated from command line to the default scheduler constructor - in other words, that `merge_parameters()` call at the beginning of the base `Scheduler` constructor won't add pick `config_id` from the global config

Reply:
Think this was just an outdated comment.  It should be back already.